src/server/scripts/Spells/spell_priest.cpp

```diff
@@ -1241,6 +1243,69 @@ class spell_pri_vampiric_touch : public SpellScriptLoader
         }
 };
 
+// 1706 - Levitate
+class spell_pri_levitate : public SpellScriptLoader
+{
+public:
+    spell_pri_levitate() : SpellScriptLoader("spell_pri_levitate") { }
+
+    class spell_pri_levitate_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_pri_levitate_SpellScript);
+
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            if (!sSpellMgr->GetSpellInfo(SPELL_PRIEST_LEVITATE) ||
```

Comment:
I suppose you can use new method to validate it, looks cleaner this way 
`return ValidateSpellInfo({ SPELL_PRIEST_LEVITATE, SPELL_PRIEST_LEVITATE_EFFECT });`